lib/email.js

```diff
@@ -6,6 +6,7 @@ var fs = require('fs');
 var contents = fs.readFileSync(__dirname+ '/email.html');
 var contents_txt = fs.readFileSync(__dirname+ '/email.txt', 'utf8');
 var email     = require('emailjs');
+var dns = require('dns');
```

Comment:
I think the 'require' line is OK ... Node.js document (https://nodejs.org/api/dns.html) says:
> Use require('dns') to access this module. 
But package.json is wrong.  See below.